tests/unit/conftest.py

```diff
@@ -116,14 +116,12 @@ def mock_airflow_connection_datarobot_jdbc(mocker, dr_jdbc_conn_details):
     mocker.patch.dict("os.environ", AIRFLOW_CONN_DATAROBOT_JDBC_DEFAULT=conn.get_uri())
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture()
 def mock_datarobot_basic_credentials(mocker, dr_basic_credentials_conn_details):
```

Comment:
```suggestion
def mock_datarobot_basic_credentials(mocker):
```

It doesn't look like you use this fixture here anymore?